### PR TITLE
Product Playbook

### PR DESCRIPTION
**Pull request by tom-mullen**
Why:

* We want to establish a clear Playbook for creating and scaling
products

This change addresses the need by:

* Creating a new section dedicated to Product

### CODE REVIEW

File: product/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Product Team
+* Chief Product Officer: Tom Mullen
+* Senior Vice President of Product: Damian Kellly
+* Vice President of Product Engineering: TBC
+
+## What We Believe In
+We believe, above all else, in the following principles, which act as guides to
+help us build incredible products:
+
+**Focus Above the Value Line**
```
Comment (ripixel) [inline product/README.md]: Under `## Strategy` you use `###` for sub-headers, but in this section you use `**text**`

File: product/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* [Product Estate](https://docs.google.com/spreadsheets/d/1YR6lM9HXHw2rkY1h2xiqOByDMlc0-1attTKJflA7A9w)
+
+### Org Structure
+![alt text](https://raw.githubusercontent.com/DVELP/cookbook/product/assets/product-org-structure.png "DVELP Product Org")
+
+## Scope of Ownership
+The product team is responsible for both home-grown `Product` and `Partner
+Solutions`. More information about each can be found below.
+
+* [Product](product)
```
Comment (ripixel) [inline product/README.md]: `cookbook/product/product` is a confusing namespace

File: product/README.md (hunk 1 above)
Comment (tom-mullen) [reply product/README.md]: Just style preferences in each of the areas.

File: product/README.md (hunk 2 above)
Comment (tom-mullen) [reply product/README.md]: Changed

File: product-org/product/products.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Products
+The following are the products that make up the `Product Estate`.
+
+### Airline
```
Comment (ripixel) [inline product-org/product/products.md:4]: I would expect to see the `Lead Project Engineer`s identified here where possible/defined

**Review by ripixel — APPROVED** [review]

**Pull request merged**